bindings/rust/s2n-tls/src/config.rs

```diff
@@ -358,6 +427,80 @@ impl Builder {
         Ok(self)
     }
 
+    /// Set a callback function that will be used to get the system time.
+    ///
+    /// The wall clock time is the best-guess at the real time, measured since the epoch.
+    /// Unlike monotonic time, it CAN move backwards.
+    /// It is used by s2n-tls for timestamps.
+    pub fn set_wall_clock_callback<T: 'static + ClockCallback>(
```

Comment:
I think it might be a good idea to have 2 distinct traits for wall clock and monotonic so implementations cannot be used interchangeably.